Source/scrollrt.cpp

```diff
@@ -679,30 +679,30 @@ void DrawItem(const Surface &out, int x, int y, int sx, int sy, bool pre)
 	if (bItem <= 0)
 		return;
 
-	ItemStruct *pItem = &Items[bItem - 1];
-	if (pItem->_iPostDraw == pre)
+	auto &pItem = Items[bItem - 1];

```

Comment:
The `p` prefix is Hungarian notation (which we are moving away from) for pointer, since this is now a reference the name should instead be just `item`.